### PR TITLE
adding Materializedviews under selfserve

### PR DESCRIPTION
**Pull request by chandugunturi**
[Preview this branch](https://cosmos-explorer-preview.azurewebsites.net/pull/1257)
**Initial screen**
image

**Provisioning screen:**
image

**Save after select provision & number of instances. It also has deleted provisioned messages:**
image

**Increasing/ decreasing number of provisioned resources:**
image

image

**Deprovisioning the resources:**
image

image

### CODE REVIEW

File: src/Localization/en/MaterializedViewsBuilder.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
+{
+    "MaterializedViewsBuilderDescription": "Provision a Materializedviews builder cluster for your Azure Cosmos DB account. Materializedviews builder is compute in your account by that performs read operations on source collection for any updates and applies them on materialized views depending on the materializedview definition.",
```
Comment (srinathnarayanan) [inline src/Localization/en/MaterializedViewsBuilder.json:2]: Add entry fopr MaterializedviewsBuilderPricing

**Comment by chandugunturi** [comment]
[MV portal changes.docx](https://github.com/Azure/cosmos-explorer/files/8505379/MV.portal.changes.docx)

**Comment by chandugunturi** [comment]
Added MVPortal changes.docx showing screen shots.

File: .vs/VSWorkspaceState.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (srinathnarayanan) [inline .vs/VSWorkspaceState.json:1]: dont check this in

Comment (srinathnarayanan) [reply .vs/VSWorkspaceState.json:1]: same for the .suo file

File: src/Localization/en/MaterializedViewsBuilder.json [hunk 3]
```diff
@@ -0,0 +1,45 @@
+{
+    "MaterializedViewsBuilderDescription": "Provision a Materializedviews builder cluster for your Azure Cosmos DB account. Materializedviews builder is compute in your account that performs read operations on source collection for any updates and applies them on materialized views as per the materializedview definition.",
```
Comment (srinathnarayanan) [inline src/Localization/en/MaterializedViewsBuilder.json:2]: Add entry for updatemessage

Comment (srinathnarayanan) [reply src/Localization/en/MaterializedViewsBuilder.json:2]: And for DeleteMessage

Comment (srinathnarayanan) [reply src/Localization/en/MaterializedViewsBuilder.json:2]: And for CreateMessage

**Review by srinathnarayanan — APPROVED** [review]

**Review by gitanuj — APPROVED** [review]

**Pull request merged**